tests/integration/test_select.py

```diff
@@ -222,3 +222,34 @@ def test_case_when_no_else(c):
     expected_df = pd.DataFrame({"C": [None, 1, 1, 1, None]})
 
     assert_eq(actual_df, expected_df)
+
+
+def test_singular_column_projection_simple(c):
+    df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
+    c.create_table("df", df)
+
+    wildcard_result = c.sql("SELECT * from df")
+    single_col_result = c.sql("SELECT b from df")
+
+    assert_eq(wildcard_result["b"], single_col_result["b"])
+
+
+@pytest.mark.parametrize(
+    "input_cols",
+    [
+        ["a"],
+        ["a", "b"],
+        ["a", "d"],
+        ["d", "a"],
+        ["a", "b", "d"],
+    ],
+)
+def test_predicate_pushdown(c, parquet_ddf, input_cols):
```

Comment:
Given that the projection step will filter out the columns anyway, it's tough to figure out if column selection is being pushed down to the IO with this test. We might need to do something similar to https://github.com/dask-contrib/dask-sql/pull/433/files to ascertain if the IO step is picking up these filters. 

Though it's non blocking and can be done in a followup pr.